core/src/main/java/org/pentaho/di/connections/vfs/provider/UndefinedConnectionFileObject.java

```diff
@@ -57,7 +57,7 @@ public FileType getType() throws FileSystemException {
 
   @Override
   public boolean isAttached() {
-    return false;
+    return true;
```

Comment:
This class was added in the context of this story. This file should better be considered attached, given there's no additional attachment needed... No major consequence.